src/components/ReturnsCalculator.jsx

```diff
@@ -0,0 +1,240 @@
+	const [expectedReturns, setExpectedReturns] = React.useState(0.0);
+	const [suggPromptsAmount]                   = useDebounce(stakeInput / 16, 0);
+	const [validatorsList, setValidatorsList]   = React.useState([]);
+	const [validatorData, setValidatorData]     = React.useState([]);
+	const [errorState, setErrorState]           = React.useState(false);
+	const [intentionData, setIntentionData]     = React.useState([]);
+	const [apiConnected, setApiConnected]       = React.useState(false);
+	const [isLoaded, setIsLoaded]               = React.useState(false);
+	const [riskLevel, setRiskLevel]             = React.useState(50);
+	const [sliderBG, setSliderBG]               = React.useState("yellow.300");
+	const ERA_PER_DAY = 4;
+
+	function suggPrompts() {
+		const data = validatorsList.map(validator => {
+			const {
+				stashId,
+				stashIdTruncated,
+				name,
+				commission,
+				totalStake,
+				poolReward,
+				noOfNominators
+			} = validator;
+			const userStakeFraction =
+				suggPromptsAmount / (suggPromptsAmount + totalStake);
+			const dailyEarning = userStakeFraction * poolReward * ERA_PER_DAY;
+			return {
+				noOfNominators,
+				stashId,
+				stashIdTruncated,
+				name,
+				commission: `${parseFloat(commission)}%`,
+				dailyEarning: isNaN(dailyEarning)
+					? "Not enough data"
+					: `${dailyEarning.toPrecision(10)} ${props.currency}`,
+				dailyEarningPrecise: isNaN(dailyEarning) ? 0 : dailyEarning
+			};
+		});
+
+		//handle suggested validaors
+		data.sort((num1, num2) => num2.dailyEarningPrecise - num1.dailyEarningPrecise);
+		const suggestedValidators = [...data.slice(0, 16)];
+		
+		const validatorInfo = {};
+		if (suggestedValidators.length > 0) {
+			// eslint-disable-next-line no-unused-vars
+			const expectedEarning = suggestedValidators.reduce((a, b) => ({
+				dailyEarningPrecise: a.dailyEarningPrecise + b.dailyEarningPrecise
+			}));
+			validatorInfo.expectedReturns = expectedEarning.dailyEarningPrecise;
+			setExpectedReturns(expectedEarning.dailyEarningPrecise);
+		}
+		validatorInfo.validatorsList = suggestedValidators;
+		validatorInfo.budget = stakeInput;
+		props.onEvent(validatorInfo);
+		
+		if (apiConnected) setIsLoaded(true);
+	}
+
+	React.useEffect(() => {
+		setValidatorsList(props.validatorData);
+	}, [props]);
+
+	if (errorState) {
+		return <ErrorMessage />;
+	}
+
+	function calculateReturns() {
+		suggPrompts();
+	}
+
+	const onRiskChange = value => {
+		console.log(value);
+		if (value <= 30) {
+			setSliderBG("green.300");
+		} else if (value > 70) {
+			setSliderBG("red.400");
+		} else {
+			setSliderBG("yellow.300");
+		}
+	};
+
+	return (
+		<React.Fragment>
+			<Helmet>
+				<title>YieldScan - Returns Calculator</title>
+				<meta name='description' content='Staking Returns Calculator' />
+			</Helmet>
+			<LogEvent eventType='Returns calculator view' />
+			<Route exact path='/returns-calculator'>
+				<Heading mt={16} mb={12} 
```

Comment:
@suyanksaraswat This is why the idea is to never skip pre-flight check -> leads to linting errors + inconsistent (at times even bad) style

Reply:
What is the issue here, can you please explain?

Reply:
Can you explain the issue here?

Reply:
Code style -> 

```<Heading mt={16} mb={12}
    ml={"calc(15% - 2rem)" // 992px upwards
    }>```

vs

```<Heading 
    mt={16}
    mb={12}
    ml={"calc(15% - 2rem)"} // 992px upwards
>```